generators/server/templates/src/test/java/package/web/rest/AccountResourceIT_skipUserManagement.java.ejs

```diff
@@ -64,8 +65,7 @@ import static <%= packageName %>.web.rest.AccountResourceIT.TEST_USER_LOGIN;
 <%_ } else { _%>
 @AutoConfigureMockMvc
 <%_ } _%>
-@WithMockUser(value = TEST_USER_LOGIN)
-@IntegrationTest
+@SpringBootTest(classes = { <%= mainClass %>.class, AsyncSyncConfiguration.class })
```

Comment:
file and class needs to be renamed for this one since its not an integration test anymore